src/Log/Engine/ConsoleLog.php

```diff
@@ -77,10 +77,11 @@ public function __construct(array $config = [])
     /**
      * Implements writing to console.
      *
-     * @param string $level The severity level of log you are making.
+     * @param mixed $level The severity level of log you are making.
```

Comment:
why mixed?

Reply:
Because of https://github.com/php-fig/log/blob/c4421fcac1edd5a324fda73e589a5cf96e52ffd0/Psr/Log/LoggerInterface.php#L116.

Reply:
That's just crazy. The truth is string|int right?
Can't we properly narrow it down?
Having mixed here is just a lie as you cant pass objects or float or alike. It is just not valid/expected in any way.

Reply:
Those are valid cases to have ignores in the static analyzer list. The base code is clearly not pragmatic/usable on this. As such we should keep it specific and adjusted to the truth for us.

Reply:
Okay, I'll revert and ignore the errors. 

Do you want to try get it changed to `int|string` for the PSR interface so that we can remove the ignore in future? :stuck_out_tongue:

Reply:
;) I trust in your politically sensitive way of approaching this.

Reply:
Currently you can only pass levels as string to our loggers. I would like to allow using `LOG_*` constants too for the `$level` argument. So let's keep the `mixed` for now and I'll make a separate PR which makes required code changes and change docblock type to `int|string`.